script.js

```diff
@@ -0,0 +1,524 @@
+let lang = localStorage.getItem('lang') || 'rus';
```

Comment:
Use constants instead of magic values
You could create variable 
` const LANGUAGE_RUS = 'rus'` and use instead of 'rus'